sdk/src/test/java/com/silanis/esl/sdk/internal/converter/DocumentPackageSettingsConverterTest.java

```diff
@@ -80,7 +74,7 @@ public void convertSDKToSDK() {
     public void convertAPIToAPI() {
 
         apiPackageSettings1 = createTypicalAPIPackageSettings();
-        apiPackageSettings2 = new DocumentPackageSettingsConverter(apiPackageSettings1).toAPIPackageSettings();
+        PackageSettings apiPackageSettings2 = new DocumentPackageSettingsConverter(apiPackageSettings1).toAPIPackageSettings();
 
         assertThat("Converter returned a null api object for a non null api object", apiPackageSettings2, notNullValue());
         assertThat("Converter didn't return the same non-null api object it was given", apiPackageSettings2, is(apiPackageSettings1));
```

Comment:
No tests have been added for the two new properties that are actually added. Please add them here.